tests/conftest.py

```diff
@@ -1594,11 +1594,9 @@ def microsalt_dir(tmpdir_factory) -> Path:
 
 
 @pytest.fixture
-def pdc_archiving_dir(
-    tmp_flow_cell_name_no_run_parameters: str, tmp_flow_cells_directory: Path, tmp_path
```

Comment:
why are there flow cell fixtures being called as tmp dirs for pdc tests? I left it as it is but this probably needs to be changed later to reduce entanglement